### PR TITLE
Feat : create error page when 404 error occurs #126

### PR DESCRIPTION
**Pull request by hummingbbird**
## Describe changes
* create error page when 404 error occurs.

## Issue number or link
#126 

## Types of changes

What is the type of code change?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (changes that resolve errors)
- [x] New feature (changes which adds functionality)
- [ ] Breaking change (big changes that affect existing functionality)
- [ ] Documentation Update

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the "README.md"
- [x] My code follows the style guidelines of this project
- [x] My changes generate no new warnings
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

_Please let me know if there's anything else to explain_

### CODE REVIEW

File: src/assets/images/curveImg.png
Comment (ancy0) [inline src/assets/images/curveImg.png:1]: 👍 
And if I use this image, I think the svg extension is better than the png.

File: src/assets/images/curved.svg
Comment (ancy0) [inline src/assets/images/curved.svg:1]: Where do you use this file?

**Review by ancy0 — APPROVED** [review]
👍

File: src/index.css [hunk 1]
```diff
@@ -11,3 +11,18 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+@font-face {
+  font-family: "IBM Medium";
```
Comment (sinji2102) [inline src/index.css:16]: 👍🏻

File: src/routes/router.js [hunk 2]
```diff
@@ -8,14 +8,15 @@ import CreateRepo from "../pages/CreateRepoPage";
...
-
+import ErrorPage from "../pages/ErrorPage";
 const Router = () => {
   return (
     <BrowserRouter>
       <Routes>
         <Route path="/" element={<MainPage />} />
         <Route path="/login" element={<LoginPage />} />
         <Route path="/select" element={<SelectTypePage />} />
+        <Route path="/*" element={<ErrorPage/>} />
```
Comment (sinji2102) [inline src/routes/router.js:19]: 😸

**Review by sinji2102 — APPROVED** [review]

File: src/assets/images/curveImg.png
Comment (hummingbbird) [reply src/assets/images/curveImg.png:1]: I tried that, but in the case of svg, there was a problem with the image shape collapsing, so I had to use png ... :(

**Pull request merged**